runtime/util/win32/openlibrary.c

```diff
@@ -308,3 +308,35 @@ j9util_open_system_library(char *name, UDATA *descriptor, UDATA flags)
 
 }
 
+char*
+util_convertSystemNUTF8PathString(J9PortLibrary* portLibrary, const char* strSrc, unsigned int encodingSrc, unsigned int encodingTarget)
```

Comment:
Any reason the port library function omrstr_convert()/j9str_convert() can't be used instead? This is what is used in many other places.

Reply:
Hmm, didn't realize it can convert between system default encode and UTF8 directly. Will give that a try.

Reply:
`omrstr_convert()` doesn't support `J9STR_CODE_MUTF8` --> `J9STR_CODE_WINDEFAULTACP/CP_ACP` conversion yet.
An OMR PR is needed.